scripts/build-and-optimize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+RUSTFLAGS="-C link-arg=-s ${RUSTFLAGS}"
+
+rm -rf "/target/"
+
+mkdir "/target/"
+
+for contract in "/code/contracts/"*
+do
+    cd "${contract}"
+
+    cargo build --release --lib --locked --target wasm32-unknown-unknown --target-dir "/target/"
+done
+
+if test $? -eq 0
+then
+    rm -rf "/artifacts/"*
+
+    for wasm in "/target/wasm32-unknown-unknown/release/"*".wasm"
+    do
```

Comment:
why don't we merge the two loops? we know each contract package produces a `<package>.wasm`.

Reply:
That is a fair point. It totally flew over my mind.